app/frontend/src/features/profile/edit/EditProfile.tsx

```diff
@@ -124,17 +130,35 @@ export default function EditProfileForm() {
     null
   );
   const queryClient = useQueryClient();
-  const { control, errors, register, handleSubmit, setValue } =
+  const { control, errors, register, handleSubmit, setValue, formState } =
     useForm<FormValues>({
       defaultValues: {
         city: user?.city,
         lat: user?.lat,
         lng: user?.lng,
         radius: user?.radius,
+        name: user?.name,
+        hometown: user?.hometown,
+        pronouns: user?.pronouns,
+        occupation: user?.occupation,
+        education: user?.education,
+        aboutMe: user?.aboutMe,
+        myTravels: user?.myTravels,
+        thingsILike: user?.thingsILike,
+        hostingStatus: user?.hostingStatus,
+        meetupStatus: user?.meetupStatus,
+        fluentLanguages: user?.languageAbilities,
+        regionsVisited: user?.regionsVisited,
+        regionsLived: user?.regionsLived,
+        additionalInformation: user?.additionalInformation,
+        avatarKey: user?.avatarKey,
       },
       shouldFocusError: true,
     });
 
+  const { isDirty } = formState;
+  setFormIsDirty(isDirty);
```

Comment:
Use `useEffect` so this isn't called every time (doesn't matter that much, but it's cleaner)